Core/clim-core/frames/frames.lisp

```diff
@@ -121,7 +121,9 @@ input focus. This is a McCLIM extension."))
    (panes-for-layout :initform nil :accessor frame-panes-for-layout
                      :documentation "alist of names and panes
                                      (as returned by make-pane)")
-
+   (resize-frame :initarg :resize-frame
+                 :initform nil
+                 :accessor frame-resize-frame)
```

Comment:
this slot is taken into account only in the change-space-requirements specialization for the top-level-sheet-pane -- appropriate code should be added also to the specialization of unmanaged-top-level-sheet-pane

Reply:
taking that into account would require handing menu-frame correcty as a frame too, so that will wait as well.